backend/curriculum_tracking/tests/test_AgileCard.py

```diff
@@ -688,3 +687,128 @@ def test_cancel_workshop_attendance(self):
         self.assertIsNone(self.card.workshop_attendance)
         self.assertEqual(WorkshopAttendance.objects.count(), 0)
         self.assertEqual(self.card.status, AgileCard.READY)
+
+
+class ReviewerIdsSinceLatestReviewRequest(TestCase):
+
+    def setUp(self):
+        self.card_1 = factories.AgileCardFactory(
+            status=AgileCard.IN_PROGRESS,
+        )
+
+        self.card_2 = factories.AgileCardFactory(
+            status=AgileCard.IN_PROGRESS,
+        )
+
+        self.project_1 = self.card_1.recruit_project
+        self.project_2 = self.card_2.recruit_project
+        self.user = self.card_1.assignees.first()
+        self.assertIsNotNone(self.card_1.assignees)
+        self.assertIsNotNone(self.card_2.assignees)
+        self.assertIsNotNone(self.card_1.recruit_project)
+        self.assertIsNotNone(self.card_2.recruit_project)
+        self.assertEqual(self.card_1.status, AgileCard.IN_PROGRESS)
+        self.assertEqual(self.card_2.status, AgileCard.IN_PROGRESS)
+        self.assertEqual(self.project_1.content_item, self.card_1.content_item)
+        self.assertEqual(self.project_2.content_item, self.card_2.content_item)
+
+        # Starting the projects (project_1 & project_2) and asserting that they are in status IN_PROGRESS
+        self.project_1.start_time = timezone.now() - timedelta(days=5)
+        self.project_2.start_time = timezone.now() - timedelta(days=5)
+        self.project_1.save()
+        self.project_2.save()
+        self.assertEqual(AgileCard.derive_status_from_project(self.project_1), AgileCard.IN_PROGRESS)
+        self.assertEqual(AgileCard.derive_status_from_project(self.project_2), AgileCard.IN_PROGRESS)
+
+        # Setting a request for review and creating four different review times (No reviews done yet)
+        self.request_review_time = self.project_1.start_time + timedelta(1)
+        self.project_1.request_review(force_timestamp=self.request_review_time)
+        self.time_one = self.project_1.start_time - timedelta(days=6)
+        self.time_two = self.project_1.start_time + timedelta(days=4)
+        self.time_three = self.project_1.start_time + timedelta(days=3)
+        self.time_four = self.project_1.start_time + timedelta(days=2)
+
+    def test_functionality_of_users_that_reviewed_since_last_review_request_for_project_1(self):
```

Comment:
please remove the word `project_1` from the test name. I shouldn't go have to look at other pieces of code to know the purpose of the test. 

`test_that_if_there_are_reviews_before_and_after_request_times_then_the_correct_ids_are_returned` might be better. It's a bit long but at least it tells us what we are trying to achieve.